Assets/Scripts/UI/Menu/ScrollingScrollView.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ScrollingScrollView : MonoBehaviour {
+
+    public float scrollSpeed = 0.5f;
+	private bool autoScroll = true;
+	private float timer = 0;
+	private float time = 0;	
+	private float waitFor = 2f;
+	public ScrollRect scrollRect;
+
+    public RectTransform rectTransform;
```

Comment:
public fields are with PascalCase. So rename to RectTransform. Same goes for ScrollRect and ScrollSpeed. Also remove this few extra empty lines, so we don't need them